liquibase/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     }
     api "io.micronaut:micronaut-management"
     api "io.micronaut.sql:micronaut-jdbc"
+    api "org.slf4j:jul-to-slf4j:1.7.30"
```

Comment:
Perhaps add this in the documentation for users instead of adding it automatically

Reply:
Removed and added to the docs